components/teleport/Teleport.vue

```diff
@@ -152,12 +158,39 @@ const {
   status,
 } = useTeleport(true)
 
+const { $i18n } = useNuxtApp()
 const { urlPrefix } = usePrefix()
 const fiatStore = useFiatStore()
 const fromChain = ref(Chain.POLKADOT) //Selected origin parachain
 const toChain = ref(Chain.ASSETHUBPOLKADOT) //Selected destination parachain
 const amount = ref() //Required amount to be transfered is stored here
 const unsubscribeKusamaBalance = ref()
+const teleportFee = ref()
+
+const nativeAmount = computed(
+  () => (amount.value || 0) * Math.pow(10, currentTokenDecimals.value),
```

Comment:
looks like it will be more sensible to giv `amount` default value of 0

Reply:
fixed